Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -200,17 +200,35 @@ def remediation_command_list_request(self, time_range: Dict[str, Any], alert_ids
     def alert_remediate_request(self, alert_id: str):
         self._http_request('PATCH', f'alert/remediation/{alert_id}', resp_type='response')
 
+    def remove_additional_resource_fields(self, input_dict, keys):
```

Comment:
It will be better if this function will show under the "HELPER FUNCTIONS" section.

Reply:
[Done](https://github.com/demisto/content/pull/32606/commits/5fa21493d916ad6da2ccfe22910449fdbc49d6d5).